ui/src/pages/topics/[id]/index.tsx

```diff
@@ -28,6 +29,13 @@ export const getServerSideProps: Types.GetServerSideProps = async (context) => {
         error = message;
     }
 
+    try {
+        const response = await api.get(`${Config.endpoints.topics}/${id}/bookmark`, token);
+        bookmark = response.data ? true : false;
```

Comment:
same as the previous comment, this should follow naming conventions for booleans